Lib/pdb.py

```diff
@@ -596,6 +602,38 @@ def _hold_exceptions(self, exceptions):
             self._chained_exceptions = tuple()
             self._chained_exception_index = 0
 
+    def _get_asyncio_loop_and_task(self):
+        try:
+            loop = asyncio.get_event_loop()
```

Comment:
You can replace this with direct call to current_task and if it exists get loop from it, using get_event_loop can have unwanted consequences

Reply:
Yeah I will change this.